src/index.ts

```diff
@@ -30,3 +31,6 @@ module.exports = function createSharedReactContext(defaultValue, key) {
   contexts[key] = context;
   return context;
 };
+
+module.exports = createSharedReactContext;
+export default createSharedReactContext;
```

Comment:
Was the a specific reason behind including `module.exports` as well?

```suggestion
export default createSharedReactContext;
```